aspnetcore/fundamentals/logging/index.md

```diff
@@ -434,12 +434,16 @@ Generally, logging should be specified in configuration and not code.
 
 ## Built-in logging providers
 
-ASP.NET Core includes the following logging providers:
+ASP.NET Core includes the following logging providers as part of the shared framework:
 
 * [Console](#console)
 * [Debug](#debug)
 * [EventSource](#event-source)
 * [EventLog](#welog)
+
+The following logging providers are shipped by Microsoft, but not as part of the 
```

Comment:
Happy to remove this sentence if any objections. But I think its best to clarify this.

Reply:
I'm okay with what's conveyed here. @Rick-Anderson might want to wordsmith this 😅